config/syslog.py

```diff
@@ -642,3 +642,57 @@ def disable_rate_limit_feature(db, service_name, namespace):
         
         if not failed:
             click.echo(f'Disabled syslog rate limit feature for {feature_name}')
+
+
+@syslog.command('level')
+@click.option("-i", "--identifier",
+              required=True,
+              help="Log identifier in DB for which loglevel is applied (provided with -l)")
+@click.option("-l", "--level",
+              required=True,
+              help="Loglevel value",
+              type=click.Choice(['DEBUG', 'INFO', 'NOTICE', 'WARN', 'ERROR']))
+@click.option("--service",
+              help="Container name to which the SIGHUP is sent (provided with --pid or --program)")
+@click.option("--program",
+              help="Program name to which the SIGHUP is sent (provided with --service)")
+@click.option("--pid",
+              help="Process ID to which the SIGHUP is sent (provided with --service if PID is from container)")
+@click.option('--namespace', '-n', 'namespace', default=None,
+              type=click.Choice(multi_asic_util.multi_asic_ns_choices()),
+              show_default=True, help='Namespace name')
+@clicommon.pass_db
+def level(db, identifier, level, service, program, pid, namespace):
+    """ Configure log level """
+    if program and not service:
+        raise click.UsageError('--program must be specified with --service')
+
+    if service and not program and not pid:
+        raise click.UsageError('--service must be specified with --pid or --program')
+
+    if not namespace:
+        cfg_db = db.cfgdb
+    else:
+        asic_id = multi_asic.get_asic_id_from_name(namespace)
+        service = f'{service}{asic_id}'
+        cfg_db = db.cfgdb_clients[namespace]
+
+    cfg_db.mod_entry('LOGGER', identifier, {'LOGLEVEL': level})
+    if not service and not program and not pid:
+        return
+
+    log_config = cfg_db.get_entry('LOGGER', identifier)
+    require_manual_refresh = log_config.get('require_manual_refresh')
```

Comment:
@Junchao-Mellanox Who add this configuration to config DB?

Reply:
Python logger will add this to config DB in __init__ function. Check this PR https://github.com/sonic-net/sonic-buildimage/pull/19611/files line 62